packages/cycle-scripts-one-fits-all/template/src/components/speaker.tsx

```diff
@@ -34,39 +34,21 @@ export function Page2(sources : Sources): Sinks {
     };
 }
 
-function intent(DOM: DOMSource) : Stream<Reducer> {
+function intent(DOM : DOMSource) : Stream<Reducer> {
     //const init$: Stream<Reducer> = xs.of<Reducer>(p => ({ count: 10 }))
-    const init$ = xs.of(
-        (prevState : State): State =>
-            typeof prevState === 'undefined' ? { count: 30 } : prevState
+    const init$ = xs.of<Reducer>(
+        prevState => (prevState === undefined ? defaultState : prevState)
     );
 
-    const add$ : Stream<Reducer> = DOM.select('.add')
-        .events('click')
-        .mapTo<Reducer>(state => ({ ...state, count: state.count + 1 }));
-
-    const subtract$ : Stream<Reducer> = DOM.select('.subtract')
-        .events('click')
-        .mapTo<Reducer>(state => ({ ...state, count: state.count - 1 }));
-
-    return xs.merge(init$, add$, subtract$);
+    return xs.merge(init$);
 }
 
-function view(state$ : Stream<State>) : Stream<VNode> {
-    return state$.map(s => s.count).map(count =>
+function view(state$: Stream<State>) : Stream<VNode> {
+    return state$.mapTo(
```

Comment:
I just need to ensure state is kept in sysnc correctly with changes and DOM refreshes. I recall some ways don;t work.